operations/helm/charts/mimir-distributed/RELEASE.md

```diff
@@ -19,80 +19,142 @@ Weekly releases have the version `x.y.z-weekly.w`, for example `3.1.0-weekly.196
 
    > **Note:** If there are any deprecated features that should be removed in this release, then verify that they have been removed, and move their deprecation notices into the section for this release.
 
-   b. Have the PR reviewed by a maintainer.
+   - Have the PR reviewed by a maintainer.
+
+   - Merge the PR upon approval.
+
+1. Create a release branch.
+
+   - Create (if the branch is not created yet), switch to and push a branch starting from the commit created by the prepare changelog PR and name it `mimir-distributed-release-x.y`.
+
+     For example, `mimir-distributed-release-4.5` for any `4.5.x` release.
+
+   - Push the branch to origin without any commit added.
+
+   - Once the branch is pushed, all changes to `mimir-distributed-release-x.y` branch must be done through PR.
+
+1. Create a branch from release branch if it hasn't been created yet, to update Mimir/GEM image and helm chart version .
+
+   For example `user/update-mimir-distributed-release-x.y`.
+
+1. Update versions and links in the `user/update-mimir-distributed-release-x.y` branch.
+
+   - Set the image versions in [values.yaml](https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/values.yaml), as needed:
+
+     - `image.tag` (Mimir)
+     - `enterprise.image.tag` (GEM)
+
+       > **Note:** Unlike the Mimir image tags, GEM image tags start with `v`. For example, `v2.6.0` instead of `2.6.0`.
+
+     - `smoke_test.image.tag` (Smoke test; usually the same as Mimir)
+     - `continuous_test.image.tag` (Continuous test; usually the same as Mimir)
+
+   - Set the `version` field, in the [Chart.yaml](https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/Chart.yaml) file, to the desired release candidate version.
+
+     For example, `4.5.0-rc.0`.
+
+     > **Note:** Once this change is merged to `mimir-distributed-x.y` branch, it will trigger the release process GitHub Action. Unless you want to release final release, make sure to append the correct rc version.
+
+   - Set the `appVersion` field, in the [Chart.yaml](https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/Chart.yaml) file, to the version of Mimir that the Helm chart deploys.
 
-   c. Merge the PR upon approval.
+     For example, `2.6.0`.
 
-   d. Create and push a branch starting from the commit created by the previous PR and name it `mimir-distributed-release-x.y`.
+   - Create or update the release notes in [release notes](https://github.com/grafana/mimir/tree/mimir-distributed-release-4.2/docs/sources/helm-charts/mimir-distributed/release-notes).
 
-   For example, `mimir-distributed-release-4.5` for any `4.5.x` release.
+     The release notes should refer to the correct Mimir and GEM versions and their specific documentation version.
```

Comment:
Add a `Note` that this step can be done in a separate PR and delayed until final release if it takes a long time.